src/py/flwr/server/strategy/fedtrimmedavg.py

```diff
@@ -0,0 +1,136 @@
+from logging import WARNING
+from typing import Callable, Dict, List, Optional, Tuple, Union
+
+from flwr.common import (
+    FitRes,
+    MetricsAggregationFn,
+    NDArrays,
+    Parameters,
+    Scalar,
+    ndarrays_to_parameters,
+    parameters_to_ndarrays,
+)
+from flwr.common.logger import log
+from flwr.server.client_proxy import ClientProxy
+
+from .aggregate import aggregate_trimmed_avg
+from .fedavg import FedAvg
+
+WARNING_MIN_AVAILABLE_CLIENTS_TOO_LOW = """
+Setting `min_available_clients` lower than `min_fit_clients` or
+`min_evaluate_clients` can cause the server to fail when there are too few clients
+connected to the server. `min_available_clients` must be set to a value larger
+than or equal to the values of `min_fit_clients` and `min_evaluate_clients`.
+"""
+
+
+# flake8: noqa: E501
+class FedTrimmedAvg(FedAvg):
+    # pylint: disable=too-many-arguments,too-many-instance-attributes,line-too-long
+    def __init__(
+        self,
+        *,
+        fraction_fit: float = 1.0,
+        fraction_evaluate: float = 1.0,
+        min_fit_clients: int = 2,
+        min_evaluate_clients: int = 2,
+        min_available_clients: int = 2,
+        evaluate_fn: Optional[
+            Callable[
+                [int, NDArrays, Dict[str, Scalar]],
+                Optional[Tuple[float, Dict[str, Scalar]]],
+            ]
+        ] = None,
+        on_fit_config_fn: Optional[Callable[[int], Dict[str, Scalar]]] = None,
+        on_evaluate_config_fn: Optional[Callable[[int], Dict[str, Scalar]]] = None,
+        accept_failures: bool = True,
+        initial_parameters: Optional[Parameters] = None,
+        fit_metrics_aggregation_fn: Optional[MetricsAggregationFn] = None,
+        evaluate_metrics_aggregation_fn: Optional[MetricsAggregationFn] = None,
+        proportiontocut: float = 0.2,
```

Comment:
I propose to call it beta, so that we stick to the naming convention of the original paper

Reply:
Definitely more descriptive, but it is common to use scientific notation in such python libraries, Flower included.

See for example: 
- https://github.com/adap/flower/blob/60a47015219913600880940d6a2402fc30e186f4/src/py/flwr/server/strategy/fedadagrad.py
- https://github.com/adap/flower/blob/60a47015219913600880940d6a2402fc30e186f4/src/py/flwr/server/strategy/fedadam.py

P.S. As for the other strategy implementations, the original paper should be cited and it is better to keep the original name or the common one used in literature. So, the class name becomes TrimmedMean along with the file name. 

However these are just details that improve the PR, you are doing a good job overall IMHO :)